### ISSUE TITLE
Undo shortcut not working

### ISSUE THREAD

**Issue by iamdannywyatt** [DISTRHO/Cardinal#511]
I don't know if this is an issue with Logic only, but when I try the Undo shortcut CMD+Z what it does is that it removes Cardinal from the slot, instead of undoing what I did inside Cardinal

**Comment by dromer** [comment]
Sounds like Logic is catching this key combo and preventing Cardinal from reading it.
Perhaps there is some setting that allows special key combos to be passed through to plugins?

**Comment by iamdannywyatt** [comment]
> Perhaps there is some setting that allows special key combos to be passed through to plugins?
You mean inside Logic? If so, there isn't one (not that I know if, at least...)

**Comment by dromer** [comment]
Yes, it's the DAW that is handling key events and seems to be catching these special key combos.
Some DAWs will have settings to allow raw key events to be sent to the plugin.

**Comment by iamdannywyatt** [comment]
Ok I just noticed something: if I go to Edit > Delete Undo History (inside Logic itself), the Undo inside Cardinal works as expected. As soon as I do something such as adding a new MIDI region and I use Cardinal's Undo, it stops working again and hitting Undo actually remove the MIDI region.

**Comment by mpicard** [comment]
This issue is also happening in Ableton Live 12.0.1 with Cardinal 23.10 on MacOS Sonoma
